app/src/main/java/com/example/hungrywolfs/adapters/BindingAdapters.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.hungrywolfs.adapters
+
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.net.toUri
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import com.example.hungrywolfs.R
+import com.example.hungrywolfs.network.MealInfo
+
+@BindingAdapter("strMealThumb")
+fun bindCircleImage(imgView: ImageView, imgUrl: String?) {
+    imgUrl?.let {
```

Comment:
SMALL suggestion. To reduce the indentation of this method, you could remove the imgUrl?.let block and use the **imgUrl ?: return** as the first line of the method.